### PR TITLE
LPS-127198 update schema version

### PR DESCRIPTION
**Pull request by marcosapmf**
Related issue: https://issues.liferay.com/browse/LPS-127198

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v4_3_2/UpgradeDDMTemplate.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+			PreparedStatement ps2 =
+				AutoBatchPreparedStatementUtil.concurrentAutoBatch(
+					connection,
+					"update DDMTemplate set script = ? where templateId = ?")) {
+
+			ps1.setLong(1, PortalUtil.getClassNameId(DDMStructure.class));
+
+			try (ResultSet rs = ps1.executeQuery()) {
+				while (rs.next()) {
+					ps2.setString(1, _convertScript(rs.getString("script")));
```
Comment (jeyvison) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v4_3_2/UpgradeDDMTemplate.java:54]: _convertScript Inline

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v4_3_2/UpgradeDDMTemplate.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+				AutoBatchPreparedStatementUtil.concurrentAutoBatch(
+					connection,
+					"update DDMTemplateVersion set script = ? where " +
+						"templateVersionId = ?")) {
+
+			ps1.setLong(1, PortalUtil.getClassNameId(DDMStructure.class));
+
+			try (ResultSet rs = ps1.executeQuery()) {
+				while (rs.next()) {
+					ps2.setString(1, _convertScript(rs.getString("script")));
```
Comment (jeyvison) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v4_3_2/UpgradeDDMTemplate.java:80]: _convertScript Inline

**Review by jeyvison — CHANGES_REQUESTED** [review]
Hi @marcosapmf , Please , see my comments :)

**Comment by marcosapmf** [comment]
@jeyvison, thanks for reviewing this pull! I've just sent a force pull with the changes you asked. Could you please check it again when you have a chance?

Thanks! 😊

**Comment by jeyvison** [comment]
ci:forward

**Review by jeyvison — APPROVED** [review]
LGTM

**Comment by marcelabc** [comment]
ci:forward

**Review by mateussgn — APPROVED** [review]

**Comment by mateussgn** [comment]
LGTM

**Pull request closed without merging**